### PR TITLE
BigAnimal: reducing privileges 

### PR DESCRIPTION
**Pull request by ccestes**
## What Changed?

As per [UPM-9904](https://enterprisedb.atlassian.net/browse/UPM-9904)

- updated Connecting to your cloud topic with permissions info

### CODE REVIEW

File: product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx [hunk 1]
```diff
@@ -26,6 +26,10 @@ Before connecting to your cloud,
     - arn:aws:iam::aws:policy/IAMFullAccess
 
     - arn:aws:iam::aws:policy/ServiceQuotasFullAccess
+
+   BigAnimal uses a custom role to provision infrastructure in your AWS cloud account. This role includes permissions for:
```
Comment (dhilipkumars) [inline product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx:30]: we dont do that yet

File: product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx [hunk 2]
```diff
@@ -35,14 +39,19 @@ Before connecting to your cloud,
 
     - Privileged Role Administrator
 
+   BigAnimal uses a custom role to provision infrastructure in your Azure cloud account. This role includes permissions for:
```
Comment (dhilipkumars) [inline product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx:42]:
```suggestion
   BigAnimal creates and uses a custom role called `biganimal-poweruser` to provision and manage infrastructure in your Azure cloud account. This role includes permissions for:
```

**Review by dhilipkumars — COMMENTED** [review]
Quick comments.

File: product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx [hunk 3]
```diff
@@ -30,8 +30,23 @@ Before connecting to your cloud,
       - bash shell version 4.0 or above.
 
       - [jq](https://stedolan.github.io/jq/) JSON parser.
+      
       - BigAnimal CLI version 1.4 or later. For detailed steps, see [Installing the CLI](../../reference/cli/#installing-the-cli).
 
+ - Be aware that BigAnimal requires permissions to run Kubernetes cluster services for PostgreSQL workloads and the associated storage services. It requires a set of supporting permissions:
+
+    - To log and monitor services to produce service telemetry information.
```
Comment (l0r3bo) [inline product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx:38]: Not a native speaker, so please disregard if I get this completely wrong. "To log and monitor services" sounds to me like BigAnimal is doing the action of logging and monitoring. While it's "services for monitoring and logging". It's a subtlety, I tried to convey the latter with the original take.

**Review by l0r3bo — APPROVED** [review]
One comment, definitely not a blocker, lgtm.

File: product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx (hunk 3 above)
Comment (ccestes) [reply product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx:38]: You're right, I didn't understand what you meant on the first go around. I'll make that change.

**Review by dhilipkumars — APPROVED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**